File: tests/openmp_examples/program_control/declare_variant.1.f90
!$omp        declare variant( p_vxv ) match( construct={parallel} )
!$omp        declare variant( t_vxv ) match( construct={target}   )
!$omp       do
!$omp       declare target
!$omp       distribute simd
!$omp    parallel
!$omp    end parallel
!$omp    target teams map(to: v1,v2) map(from: v3)
!$omp    end target teams

File: tests/openmp_examples/program_control/declare_variant.1.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp declare variant( p_vxv ) match( construct={parallel} )
#pragma omp declare variant( t_vxv ) match( construct={target} )
#pragma omp for
#pragma omp begin declare target
#pragma omp distribute simd
#pragma omp end declare target
#pragma omp parallel
#pragma omp target teams map(to: v1[:100],v2[:100]) map(from: v3[:100])
